graphql/resolvers.go

```diff
@@ -942,6 +942,21 @@ func (r *queryResolver) Patch(ctx context.Context, id string) (*restModel.APIPat
 	if err != nil {
 		return nil, InternalServerError.Send(ctx, err.Error())
 	}
+	failedAndAbortedStatuses := append([]string{evergreen.TaskAborted}, evergreen.TaskFailureStatuses...)
```

Comment:
nit: a cleaner append would be `append(evergreen.TaskFailureStatuses, evergreen.TaskAborted)`